.pre-commit-config.yaml

```diff
@@ -36,6 +36,11 @@ repos:
       args: [ "-w", "-s" ]
 - repo: local
   hooks:
+    - id: protected-branches
+      name: protected-branches
+      description: checks that the commit isn't created on a protected branch
+      entry: "python3 tasks/git-hooks/protected-branches.py"
```

Comment:
```suggestion
      entry: 'python3 tasks/git-hooks/protected-branches.py'
```

Reply:
done, is that more of a convention or is there a technical difference?

Reply:
Yes ! My second name is linter 😎

Reply:
So you're picky, slow on windows, and use a lot of memory, gotcha :grin: